php7.1-fpm/skilld-php-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Optionally enable Xdebug.
+if [[ $PHP_XDEBUG_ENABLED = 1 ]]; then
+  line="zend_extension=$(php-config --extension-dir)/xdebug.so"
+  ini="/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini"
```

Comment:
I think better have no such file, cos xdebug should be compiled in Dockerfile (I see it's not yet) and later enabled with changing CMD `php-fpm7 -F -d zend_extension=xdebug.so`